Sources/HttpServer/Actions/UrlDecodeAction.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+public class UrlDecodeAction: HttpAction {
+    private var routesTable: RoutesTable
+    public var dataStorage: DataStorage
+
+    public init(routesTable: RoutesTable, dataStorage: DataStorage) {
+        self.dataStorage = dataStorage
+        self.routesTable = routesTable
+    }
+
+    public func execute(request: HttpRequest) -> HttpResponse {
+        let decodedString = decodeEachParamOfARequest(params: request.returnParams())
+                            .replacingOccurrences(of: "1=O", with: "1 = O")
+                            .replacingOccurrences(of: "2=s", with: "2 = s")
```

Comment:
Is it possible to generalize this? Instead of specifically targeting "1=0" and "2=s", can you just do "<any number>=<any character>"?

Reply:
There are other "=" in the param string like "!=". So I cannot just convert all "=" to " = "